tests/packagedcode/test_plugin.py

```diff
@@ -137,6 +137,13 @@ def test_package_command_scan_phpcomposer(self):
         run_scan_click(['--package', '--strip-root', '--processes', '-1', test_dir, '--json', result_file])
         check_json_scan(expected_file, result_file, regen=False)
 
+    def test_package_command_scan_podspec(self):
+        test_dir = self.get_test_loc('podspec/package')
```

Comment:
Try to avoid reusing the same test files in a different test as this creates unnecessary coupling between tests that will bite us in the future.

Here you want to have a dedicated directory for this test f=data files and its expected file.